packages/recommend-js/src/types/html.tsx

```diff
@@ -0,0 +1,5 @@
+import { VNode } from '@algolia/recommend-vdom';
```

Comment:
You can actually just use `import { html } from 'htm/preact` which [simply returns a `VNode`](https://github.com/developit/htm/blob/d62dcfdc721e47bc1923a2cb7a01ebd594ab0c25/src/integrations/preact/index.d.ts#L4)

Reply:
Which means this solves the issue with `VNode | VNode[]` :)

Reply:
I noticed that in autocomplete, we use this VNode | VNode[]. That is why I wanted to use VNode[], to have the same type accross our lib. In. the meantime you are right and it does solve our issue. 🤔

Reply:
Indeed in autocomplete templates are considered just as functions that return vdom elements.
In recommend we can pass components that will be instantiated via `createElement` or `h` depending on whether we used React or Preact, which I think is even better as it allows better debugging and usage of hooks 😄